TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Mqs/DeleteMq/CheckAnswers.cshtml

```diff
@@ -45,16 +35,29 @@
                     <govuk-summary-list-row-key>End date</govuk-summary-list-row-key>
                     <govuk-summary-list-row-value data-testid="end-date">@(Model.EndDate.HasValue ? Model.EndDate.Value.ToString(UiDefaults.DateOnlyDisplayFormat) : "None")</govuk-summary-list-row-value>
                 </govuk-summary-list-row>
+            </govuk-summary-list>
+            <h2 class="govuk-heading-m">Why are you deleting this mandatory qualification?</h2>
+            <govuk-summary-list data-testid="change-reason-summary">
+                <govuk-summary-list-row>
+                    <govuk-summary-list-row-key>Reason</govuk-summary-list-row-key>
+                    <govuk-summary-list-row-value data-testid="deletion-reason">@Model.DeletionReason!.GetDisplayName()</govuk-summary-list-row-value>
+                </govuk-summary-list-row>
+                <govuk-summary-list-row>
+                    <govuk-summary-list-row-key>Additional information</govuk-summary-list-row-key>
+                    <govuk-summary-list-row-value data-testid="deletion-reason-detail">
+                        @Html.ConvertNewlinesToLineBreaks(!string.IsNullOrEmpty(Model.DeletionReasonDetail) ? Model.DeletionReasonDetail : "None")
+                    </govuk-summary-list-row-value>
+                </govuk-summary-list-row>
                 <govuk-summary-list-row>
                     <govuk-summary-list-row-key>Evidence</govuk-summary-list-row-key>
                     <govuk-summary-list-row-value data-testid="evidence">
                         <vc:evidence-file-link evidence-file="@Model.EvidenceFile" />
                     </govuk-summary-list-row-value>
                 </govuk-summary-list-row>
             </govuk-summary-list>
-
+            <h2 class="govuk-heading-m">Are you sure you want to delete this mandatory qualification?</h2>
             <div class="govuk-button-group">
-                <govuk-button class="govuk-button--warning" type="submit">Delete qualification</govuk-button>
+                <govuk-button class="govuk-button--warning" type="submit">Confirm and delete qualification</govuk-button>
```

Comment:
Should I be adding something to group the new heading with the buttons for accessibility?